config/config.exs

```diff
@@ -1 +0,0 @@
-use Mix.Config
```

Comment:
This part I'm not 100% sure about. It looks to me like this library doesn't need a config but I might be mistaken.